examples/global-external-auth/01-authserver.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testserver
+  labels:
+    app.kubernetes.io/name: testserver
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: testserver
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: testserver
+    spec:
+      containers:
+      - name: testserver
+        image: docker.io/projectcontour/contour-authserver:v2
+        imagePullPolicy: IfNotPresent
+        command:
+        - /contour-authserver
+        args:
+        - testserver
+        - --address=:9443
+        ports:
+        - name: auth
+          containerPort: 9443
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 100m
+            memory: 30Mi
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: testserver
```

Comment:
this'll need a `namespace: projectcontour` as well since that's where the ExtensionService is..